test/e2e/init_scale_zero_test.go

```diff
@@ -56,6 +56,9 @@ func TestInitScaleZeroServiceLevel(t *testing.T) {
 	}
 	defer test.TearDown(clients, names)
 
+	updatedConfigAutoscalerCM := setCheckValidityOnDeployOnCluster(t, clients, "config-autoscaler", false)
+	defer restoreCM(t, clients, updatedConfigAutoscalerCM)
```

Comment:
need to do it for when test is interrupted too.

Reply:
the interrupt is handled inside `setCheckValidityOnDeployOnCluster(..)`